Fix dbt mention (#9681)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/dbt-platform-release-notes-gen.md

### dbt platform

- **AI providers settings page consolidated**: The Copilot and Wizard settings pages are unified under a single "AI providers" page at `/settings/accounts/{id}/pages/ai`. The previous `/pages/copilot` URL redirects automatically, and the sidebar item and page title now use "AI providers."

> - **"Enable dbt State" checked by default on job create**: When creating a new job, the **Enable dbt State** checkbox is now checked by default on both dbt and Fusion environments when dbt State is available and an active subscription is present.

Reviewer: ```suggestion
- **"Enable dbt State" checked by default on job create**: When creating a new job, the **Enable dbt State** checkbox is now checked by default on all environments when dbt State is available and an active subscription is present.
```

> - **"Enable dbt State" checked by default on job create**: When creating a new job, the **Enable dbt State** checkbox is now checked by default on both dbtt and Fusion environments when dbt State is available and an active subscription is present.

Author: ```suggestion
- **"Enable dbt State" checked by default on job create**: When creating a new job, the **Enable dbt State** checkbox is now checked by default on both dbt and Fusion environments when dbt State is available and an active subscription is present.
```
